steveDevonport/steve_devonport.md

```diff
@@ -0,0 +1,46 @@
+
+After a couple of sprints pass by uneventful I take a look at our last Sprint velocity and note that it has shot up dramatically as stories have been brought forward from downstream sprints, this surprised me as I don't remember anyone in any of the ceremonies mentioning it. I mention this change in our velocity during the next retrospective and our new engineer (I think we'll call him Steve) Steve says that he decided to pick stories that he wanted to work on. I looked at the team manager and he shrugged his shoulders and said, 'As Steve is learning about our systems I thought it would be a good learning experience for him.' There was a 'healthy' discussion that consumed the rest of the retrospective!
+
+As time passed and I began building out the Test Automation frameworks, ready for the team to inherit and populate, and the presentations for how we can start to reduce the size of our deliveries into production, I logged on on one evening to get some company branding and noted that Steve was online. So I started a conversation with him. As part of that conversation it transpired that he was working 2-3 nights every week and sometimes the weekend, 'it's the only way I can get the work done' was his defence.
+
+It didn't take too long before other team members start working the odd evening or two in the week and once there were 4 engineers working the weekend.
+
+Naturally this approach broke apart any notion of completing our transition to being a true Agile delivery team.
+
+I spoke to the team manager, privately, about this at some length, stressing how anti-team Steve's approach is. The team manager didn't take too kindly to my 'attack' on his buddy and I was eventually told, "don't be so negative, we're getting the work done aren't we?"
+
+I couldn't deny what the team manager had said to me because it was true we were delivering features into production faster than before and quality was decent enough. But there was no team spirit anymore, people were working crazy hours sometimes starting at 6 or finishing at 10pm, plus weekends. Everyone was getting burned out, especially me as I was writing automated tests for several web services that were being released as a 'big bang'.
+
+This was not how I wanted to work, I was not enjoying how the team was falling back on old ways of working.
+
+As we made the big push, over the weekend, into production and our components came online...nothing worked! With all the rushing and pushing it seems we had missed several integration points across the components...Anyway it took us about a week of head scratching, rework, and long hours going through the server logs to isolate the missing items.
+
+It wasn't too long after that I resigned.
```

Comment:
What did you learn from this whole experience?  If you did it again, would you change anything?

Reply:
This one is quite funny, I've got a new role, at a different company, based off the above experience - so I've learnt lots. I'll remove that last line and change the emphasis to the positive outcome.